demos/python_demos/3d_segmentation_demo/README.md

```diff
@@ -66,13 +72,17 @@ python3 3d_segmentation_demo.py -i <path_to_image>/inputImage.tiff -m <path_to_m
 For example, to do inference on 3D NIfTI images using a trained network with multiple outputs on CPU and save 
 output TIFF and NIFTI images, run the following command:
 ```
-python3 3d_segmentation_demo.py -i <path_to_nifti_images> -m <path_to_model>/brain-tumor-segmentation-0001 -d CPU -o <path_to_output> -nii
+python3 3d_segmentation_demo.py -i <path_to_nifti_images> -m <path_to_model>/brain-tumor-segmentation-0001 -d CPU -o <path_to_output> -nii -ms 2,0,3,1
 ```
-
 For example, to do inference on a single 3D NIfTI image and save an output TIFF image, run the following command:
 ```
-python3 3d_segmentation_demo.py -i <path_to_nifti_image>/PackedImage.nii -m <path_to_model>/brain-tumor-segmentation-0001 -d CPU -o <path_to_output>
+python3 3d_segmentation_demo.py -i <path_to_nifti_image>/PackedImage.nii -m <path_to_model>/brain-tumor-segmentation-0001 -d CPU -o <path_to_output> -ms 2,0,3,1
+```
+`-ms` option aligns input modalities that depend on a dataset. For example, [Medical Decathlon](http://medicaldecathlon.com/) brain tumor segmentation data modalities follow in different order than it's required by nets. To make a correct order using Medical Decathlon brain tumor data the correct option is `2,0,3,1` for `brain-tumor-segmentation-0001` and `1,2,3,0` for `brain-tumor-segmentation-0002`.
```

Comment:
If the two models take the inputs in different order, why does the help message for the `-ms` option list a single order?

Reply:
It contains order which is used in demo, a user is required to check correct dataset order and transfer it to the order from a help message

Reply:
But this paragraph talks about a single dataset and yet there are two different orders.

Reply:
Please, check a new rephrasing of a help message

Reply:
It's better, but you need to also update the README and the exception message in `mri_sequence`.